pkg/ebpf/c/types.h

```diff
@@ -26,6 +26,8 @@ typedef struct task_context {
     u64 parent_start_time;        // parent process task leader's monotonic start time
 } task_context_t;
 
+typedef u16 policies_version_t;
```

Comment:
This is a no IMO. Why to add complexity (besides a huge var name).

Reply:
FWIW, I prefer the explicitness, not sure how relevant the type change will actually be (if we will need to change it).

Reply:
Ok, I'll revert this. Checked again and noticed that we only use typedef for structs but https://github.com/aquasecurity/tracee/blob/f3fa64f0d3bd8c8203dd8199c182375f78d295ec/pkg/ebpf/c/types.h#L540.